preliz/distributions/continuous.py

```diff
@@ -662,6 +662,77 @@ def rvs(self, size=1, random_state=None):  # pylint: disable=arguments-differ
         return np.abs(self.dist.rvs(size=size, random_state=random_state))
 
 
+class InverseGamma(Continuous):
+    r"""
+    Inverse gamma log-likelihood, the reciprocal of the gamma distribution.
+
+    The pdf of this distribution is
+
+    .. math::
+
+       f(x \mid \alpha, \beta) =
+           \frac{\beta^{\alpha}}{\Gamma(\alpha)} x^{-\alpha - 1}
+           \exp\left(\frac{-\beta}{x}\right)
+
+    .. plot::
+        :context: close-figs
+
+        import arviz as az
+        from preliz import InverseGamma
+        az.style.use('arviz-white')
+        alphas = [1., 2., 3.]
+        betas = [1., 1., .5]
+        for alpha, beta in zip(alphas, betas):
+            InverseGamma(alpha, beta).plot_pdf()
```

Comment:
This plot looks weird compared with the one from PyMC. Try with

```suggestion
            InverseGamma(alpha, beta).plot_pdf(support=(0, 3))
```